shared.js

```diff
@@ -92,6 +93,16 @@ var Shared = module.exports = {
 		case "v8>":
 			// context.channel.send_reply(context.intent, "v8 temporarily disabled, please use js> instead."); return;
 			engine = Sandbox.V8; break;
+		case "n>":
+		case "node>":
+		case "0.10>":
+		    engine = Sandbox.Node; break;
+		case "nh>":
+		    engine = Sandbox.NodeHarmony; break;
+		case "babel>":
+		case "b>":
+		case "6>":
```

Comment:
this will conflict with a future node version - maybe `es6` would be a better prompt?